install_script.sh.template

```diff
@@ -51,6 +51,9 @@ exec 1>&-
 exec 1>$npipe 2>&1
 trap 'rm -f $npipe' EXIT
 
+##
+# REPORTING AND COMMON METHODS
+#
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Based on your other comments
```suggestion
##
```